roles/registry/defaults/main.yml

```diff
@@ -1,8 +1,20 @@
-images:
+china: False
+images_world:
   - { url: "quay.io/coreos/etcd:v3.2.15", name: "etcd"}
   - { url: "gcr.io/google_containers/hyperkube:v1.8.4", name: "hyperkube"}
   - { url: "gcr.io/google_containers/pause-amd64:3.0", name: "pause"}
   - { url: "quay.io/calico/node:v1.2.1", name: "node"}
   - { url: "quay.io/calico/cni:v1.8.3", name: "cni"}
   - { url: "quay.io/calico/kube-policy-controller:v0.6.0", name: "kube-policy-controller"}
   - { url: "quay.io/calico/ctl:v1.2.1", name: "ctl"}
+
+images_cn:
+  - { url: "registry.cn-hangzhou.aliyuncs.com/k8s0/etcd:v3.2.15", name: "etcd"}
+  - { url: "registry.cn-hangzhou.aliyuncs.com/k8s0/hyperkube:v1.8.4", name: "hyperkube"}
```

Comment:
The version is fixed to `v1.8.4`. I don't think this is friendly for other releases. I'd prefer to use `kubeadm init` handle this.

Also You can set an alternative `imageRepository` to be used instead of default `gcr.io`.

Reply:
Archi. Xu, Why don't u JUST submit a PR???